openhcl/virt_mshv_vtl/src/processor/snp/mod.rs

```diff
@@ -678,6 +678,9 @@ fn init_vmsa(vmsa: &mut VmsaWrapper<'_, &mut SevVmsa>, vtl: GuestVtl, vtom: Opti
     // Efer has a value that is different than the architectural default (for SNP, efer
     // must always have the SVME bit set).
     vmsa.set_efer(x86defs::X64_EFER_SVME);
+
+    let features = vmsa.sev_features();
+    tracing::info!(?vtl, ?features, "VMSA features");
```

Comment:
nit: can we merge this with the previous trace? Less noise and easier to find all the relevant data in one place in Kusto.